lib/pengine/unpack.c

```diff
@@ -72,15 +72,20 @@ pe_fence_node(pe_working_set_t * data_set, node_t * node, const char *reason)
         resource_t *rsc = node->details->remote_rsc->container;
 
         if (is_set(rsc->flags, pe_rsc_failed) == FALSE) {
-            crm_warn("Guest node %s will be fenced (by recovering %s) %s",
-                node->details->uname, rsc->id, reason);
+            if (!is_set(rsc->flags, pe_rsc_managed)) {
+                crm_notice("Not fencing guest node %s because the container is "
+                           "unmanaged", node->details->uname);
```

Comment:
I think it would be worthwhile to print the "otherwise would ..." clause, so the user can see what's going wrong. The text may be somewhat misleading in the cleanup case, but if there's a real reason to recover the resource, it would be good to see that.